event_sourcery_django/subscription.py

```diff
@@ -27,7 +35,13 @@ def subscribe_to_category(
         timelimit: timedelta,
         category: str,
     ) -> Iterator[list[RecordedRaw]]:
-        raise NotImplementedError
+        return GapDetectingIterator(
+            gap_retry_interval=self.GAP_RETRY_INTERVAL,
+            start_from=start_from,
+            batch_size=batch_size,
+            timelimit=timelimit,
+            filtering=Q(stream__category=category),
```

Comment:
Initially, I wanted to go through ALL events without adding SQL filters and filter them out in Python.

WDYT? Filtering in Python (I assume) will be slow when there are lots of event types and we're only interested in SOME of them. On the other hand, current implementation will almost always fall in gap, because there will be no continuous list of IDs (because some rows were filtered-out on SQL-level)

Reply:
This is an interesting topic. I can agree on whatever approach today, but it may be a configuration later. Let's stick with your approach for now.